### ISSUE TITLE
Task - Email of Hidden user is revealed in the task preview in search list

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#36060]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v1.4.38-1
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

## Action Performed:
. Go to staging.new.expensify.com
2. Go to 1:1 DM with a Hidden user.
3. Create a task without assignee in the DM.
4. Open task thread.
5. Click Assignee and select the Hidden user as assignee.
6. Return to main chat and go to task thread again.
7. Open search list.

## Expected Result:
The task preview in the search list will show "assigned to Hidden".

## Actual Result:
The task preview in the search list shows "assigned to <full_email>" instead of "assigned to Hidden".

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/93399543/17c434c3-4cfa-47f0-ac49-bf9a350a5a85

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Tony-MK** [comment]
The root cause of this issue is the same as https://github.com/Expensify/App/issues/35960.

The backend and fronten fixes should include this issue as well.

**Comment by Tony-MK** [comment]
The root cause is the same as https://github.com/Expensify/App/issues/35960.

The backend and frontend fixes should solve this issue as well.

**Comment by mananjadhav** [comment]
@s77rt Can you confirm if the linked issue fixes this one? If yes, then we can close this one out and update the issue description to add this one to the scope.

**Comment by s77rt** [comment]
@mananjadhav Yes looks same to me

**Comment by strepanier03** [comment]
Should we close this one or leave it open on hold to check the issue is fixed from #36196?

**Comment by s77rt** [comment]
Since editing the last message is intended the linked issue and PR won't fix this. I'm not sure about the expected behaviour here but I guess this need to be fixed in the BE.

@thienlnam If you assign a "Hidden" user should that user still stay hidden?

**Comment by thienlnam** [comment]
Yeah it sounds like it - if you didn't have their personal details before assigning them then they shouldn't be revealed after getting assigned

**Comment by strepanier03** [comment]
Aah so maybe this needs to be Internal then?

**Comment by strepanier03** [comment]
Switched to Internal for a BE fix.

**Comment by strepanier03** [comment]
Waiting for someone to pick this up.

**Comment by strepanier03** [comment]
Waiting for internal ownership for the fix.

**Comment by thienlnam** [comment]
All task issues are future, we can just bump the priority of this to match that

**Comment by strepanier03** [comment]
Changing this to weekly as tasks are slated for future work. Thanks for the heads up @thienlnam!

**Comment by strepanier03** [comment]
Still held for future work.

**Comment by strepanier03** [comment]
Future work.

**Comment by strepanier03** [comment]
Labeled for future work, okay for now.

**Comment by strepanier03** [comment]
Waiting for prioritization.

**Comment by mananjadhav** [comment]
Should we be putting this on Hold if not going to be prioritized?

**Comment by strepanier03** [comment]
@mananjadhav - Sorry for the hold up, I've been OoO and this hasn't been a priority. Seeing as VSB is pretty much on hold now, I think we can put it on hold for now.

**Comment by strepanier03** [comment]
I'm setting monthly, this is on long term hold.

**Comment by strepanier03** [comment]
On hold at this time.

**Comment by strepanier03** [comment]
Still held on prioritization.

**Comment by mananjadhav** [comment]
@strepanier03 Any updates on this one?

**Comment by strepanier03** [comment]
This is for Tasks and we're not actively working or developing them unless they affect the onboarding flow, this does not. I'm closing and making not a priority.
